### PR TITLE
Design v22

### PR DESCRIPTION
**Pull request by levogirar**
Refactored CSS
Cleaned JS
Styled 404 page
Styled admin/setting
Combined branch `design-v22` with `develop-v22` including #172 pull request

### CODE REVIEW

File: WebRoot/Templates/navbar admin.php [hunk 1]
```diff
@@ -5,12 +5,14 @@
...
 		'/admin/database' => ['Database', 'dns.svg'],
-		'/admin/settings' => ['Settings', 'label.svg']
+		'/admin/settings' => ['Settings', 'slider.svg']
 	];
 ?>
 <div class="sticky">
 	<div class="navbar-box" id="navbar">
-		<img class="panel-ribbon" src="/static/logo/ribbon.svg" alt="wuwana logo ribbon">
+		<div class="panel-ribbon">
+			<img src="/static/logo/ribbon.svg" alt="wuwana logo ribbon">
```
Comment (Nils85) [inline WebRoot/Templates/navbar admin.php:14]: This image should have alt="" to indicates that is not a key part of the content

File: WebRoot/Templates/navbar homepage.php [hunk 2]
```diff
@@ -1,12 +1,14 @@
 <div class="navbar-box" id="navbar">
-	<img class="panel-ribbon" src="/static/logo/ribbon.svg" alt="wuwana logo ribbon">
+	<div class="panel-ribbon">
+		<img src="/static/logo/ribbon.svg" alt="wuwana logo ribbon">
```
Comment (Nils85) [inline WebRoot/Templates/navbar homepage.php:3]: Same here

File: WebRoot/static/dhtml/main.js [hunk 3]
```diff
@@ -1,160 +1,131 @@
-// ECMAScript 5
-"use strict";
+// Global Variables
+let url = window.location.href;
+let body = document.body.style;
+let icon = document.getElementById("menu-icon");
+let winWidth = window.innerWidth;
```
Comment (Nils85) [inline WebRoot/static/dhtml/main.js:5]: 😱 No respect for this poor ES5... Please do something!

File: WebRoot/static/dhtml/main.js [hunk 4]
```diff
@@ -1,160 +1,131 @@
...
-const body = document.body;
-const icon = document.getElementById("menu-icon");
-const navbar = document.querySelector(".navbar-box");
-const bkg = document.querySelector(".navbar-background");
-const filter = document.getElementById("filter");
+window.addEventListener("resize", () => 
+{
+	let navbar = document.querySelector(".navbar-box").style;
+	let bkg = document.querySelector(".navbar-background").style;
+	let filter = document.getElementById("filter").style;
```
Comment (Nils85) [inline WebRoot/static/dhtml/main.js:22]: It's simple, to stay compatible with ES5 just replace all "let" and "const" by "var". That's it.

**Review by Nils85 — CHANGES_REQUESTED** [review]

File: WebRoot/static/dhtml/main.js (hunk 4 above)
Comment (levogirar) [reply WebRoot/static/dhtml/main.js:22]: hahahahah

File: WebRoot/static/dhtml/main.js (hunk 3 above)
Comment (levogirar) [reply WebRoot/static/dhtml/main.js:5]: hahahaha love your comments

File: WebRoot/static/dhtml/main.js [hunk 5]
```diff
@@ -1,164 +1,135 @@
-// ECMAScript 5
-"use strict";
```
Comment (Nils85) [inline WebRoot/static/dhtml/main.js:2]: These lines are necessary.

**Review by Nils85 — CHANGES_REQUESTED** [review]

File: WebRoot/static/dhtml/main.js (hunk 5 above)
Comment (levogirar) [reply WebRoot/static/dhtml/main.js:2]: Noted. I just fixed that on the PR #175

**Review by Nils85 — APPROVED** [review]

**Pull request merged**